Headers/Public/VLCMediaPlayer.h

```diff
@@ -875,6 +889,28 @@ extern NSString *const VLCTitleDescriptionIsMenu;
 @property (NS_NONATOMIC_IOSONLY, readonly) NSImage *lastSnapshot;
 #endif
 
+/**
+ * Start recording at given **directory** path
+ * \param path directory where the recording should go
+ * \return an BOOL with the success status
+ */
+- (BOOL)startRecordingAtPath:(NSString *)path;
+
+/**
+ * Stop current recording
+ * \return an BOOL with the success status
+ */
+- (BOOL)stopRecording;
```

Comment:
when can recording fail?

Reply:
Right now it returns no when there is no input stream. That doesn't seem right because you could be not recording and stopRecording would return NO as in it was not able to stop recording. Someone  from the core needs to look over the patches. I'm not sure if this is correctly implemented on the core side